main.js

```diff
@@ -0,0 +1,43 @@
+var chkBook = document.querySelector('.expenses');
+
+var addstuffList = function(event) {
+  var itemsBought = document.getElementById('expenses');
+  var costSpent = document.getElementById('amount');
+
+  var eachItem = document.createElement('li');
+  var eachBill = document.createElement('li');
+
+  eachItem.setAttribute('class', 'itemsPer');
+  eachBill.setAttribute('class', 'costsPer');
+
+  eachItem.setAttribute('contenteditable', true);
+  eachBill.setAttribute('contenteditable', true);
+
+  // agendaStuff.innerHTML=value;
+  // spentMoney.innerHTML=value;
+  document.getElementById('add-expense').onClick = function() {
```

Comment:
Look at using `addEventListener` because it'll let you register multiple event handlers. `onclick` will only support one.